pkg/llms/zhipuai/params.go

```diff
@@ -65,11 +65,13 @@ type Prompt struct {
 }
 
 func DefaultModelParams() ModelParams {
+	// TODO: should allow user to configure the temperature and top_p of inference
+	// use 0.8 and 0.7 for now
 	return ModelParams{
 		Model:       ZhiPuAILite,
 		Method:      ZhiPuAIInvoke,
-		Temperature: 0.95, // zhipuai official
-		TopP:        0.7,  // zhipuai official
+		Temperature: 0.8, // more accurate?
```

Comment:
Right. We acutally allow user to configure these confiugratoin. https://github.com/kubeagi/arcadia/blob/main/api/base/v1alpha1/prompt_types.go#L30C31-L30C31 

And 0.95 and 0.7 are the detaul params provided by ZhipuAI.